itchysats/umbrel-app.yml

```diff
@@ -60,5 +60,6 @@ defaultUsername: itchysats
 deterministicPassword: true
 torOnly: false
 releaseNotes: >-
-  This release enables ETHUSD CFDs. You can now trade BTCUSD and ETHUSD using your Bitcoin.
-  With this release support for the legacy network stack is dropped; versions prior to 0.5.4 are not supported anymore and have to upgrade to maintain compatibility.
\ No newline at end of file
+  This release fixes a minor UI bug for 0.6.0.
```

Comment:
1 tiny thing (which was also there in the previous release) is; these lines won't appear on new lines so all the text is simply concatenated. With the 3 points; can you add an empty between each line? I would do this myself, but for some reason I can push to your branch.

Reply:
Done, thanks for the hint!